swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
+    def __init__(self, category = '', condition = 0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
```

Comment:
What would happen here if the condition fell outside the expected range of values? Since condition is a float, what would happen if it were set to something like `3.5`? How could we handle either of these situations?